controllers/controllers_test.go

```diff
@@ -1,6 +1,8 @@
 package controllers
 
 import (
+	"fmt"
```

Comment:
Recommend removing fmt unless you use it for something other than print statements